core/src/main/java/org/frankframework/align/content/JsonDocumentContainer.java

```diff
@@ -146,22 +137,10 @@ protected void generate(JsonGenerator g, String key, Object item) {
 		}
 	}
 
-
 	private void newLine(StringBuilder sb, int indentLevel) {
-		if (indentLevel>=0)  {
-			sb.append(INDENTOR, 0, (indentLevel<MAX_INDENT?indentLevel:MAX_INDENT)*2+1);
+		if (indentLevel >= 0) {
+			sb.append(INDENTOR, 0, (Math.min(indentLevel, MAX_INDENT)) * 2 + 1);
 		}
 	}
 
```

Comment:
I'm somewhat doubtful still that the getter/setter for `skipRootElement` can be completely removed.

Reply:
I think it is only used and set while constructing the object.